nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/impl/AbstractProperty.java

```diff
@@ -379,16 +381,25 @@ protected boolean isDeprecated() {
         return deprecated.booleanValue();
     }
 
+    protected boolean isSecured() {
+        PropertyCharacteristicHandler propertyHandler = Framework.getService(PropertyCharacteristicHandler.class);
+        // remove prefix
+        String xPath = getXPath();
+        String path = xPath.substring(xPath.lastIndexOf(':') + 1);
```

Comment:
i know we prefer avoid import when it's not required  :),  but i was thinking about using `org.apache.commons.lang3.StringUtils#substringAfter(xPath, ":")`

Reply:
In fact we can't use `substringAfter` because as stated in javadoc:
>If nothing is found, the empty string is returned.
In our case, we can have a xpath without prefix and so without separator.

Reply:
ah ok i see :)